website/docs/01-features/index.md

```diff
@@ -110,16 +110,16 @@ Themes in Onion are two things : <br />
 <p align="center"><img src={require('./assets/activity-tracker.webp').default} style={{width: 320}} /></p>
 
 :::note <a href="apps/activity-tracker">Activity Tracker overview</a>
-Thanks to `Activity Tracker` app you can : 
-- Track your game playtimes.
-- View your total playtime in the top bar.
+Thanks to `Activity Tracker` app you can: 
+- Track your game playtimes
+- View your total playtime in the top bar
 - Share your playtimes by taking a screenshot (press <kbd>MENU</kbd>+<kbd>POWER</kbd> - screenshot is saved in `Screenshots` folder).
```

Comment:
You've removed the dot in 113-115, but not in 116, was that on purpose?

Reply:
I missed it!